draft-rbw-home-servers.md

```diff
@@ -206,25 +218,27 @@ R-REVOKE-AUTH:
 
 ## Delegated Credentials {#delegated}
 
-Delegated credentials {{?RFC9345}} allows the entity operating the CPE
-(e.g., vendor or ISP) to sign a 7-day validity for the CPE's public key.
-The frequency of CA interactions remains the same as with normal
-certificates ({{normal-certificates}}), but the interactions are with
-the vendor's service rather than the public CA.
-
-As currently specified in {{?RFC9345}}, the same name would be issued to all CPEs
-making it impossible to identify whether the delegated credential is
-issued to the intended CPE or an "evil-twin" CPE. This drawback can be corrected
-by enhancing {{?RFC9345}} to include a string that uniquely identifies the
-delegated credential (e.g., including hash of customer id or other unique identifier
-in the FQDN such as "HASH.cpe.example.com").
+Delegated credentials {{?RFC9345}} allows the entity operating the
+device (e.g., vendor or ISP) to sign a 7-day validity for the device's
+public key (Short-Term Automatically Renewed (STAR)).  The frequency of CA interactions remains the same as with
```

Comment:
I don't see any mention of the term STAR in RFC9345. You seem to have confused with https://www.rfc-editor.org/rfc/rfc8739.html

Reply:
I agree STAR is not mentioned in RFC9345.  However, RFC9345 says this in its Section 3,

> To limit exposure in case of the compromise of a delegated credential's private key, delegated credentials have a maximum validity period. In the absence of an application profile standard specifying otherwise, the **maximum validity period is set to 7 days**. 

which meets all the meaning of STAR!  

I will soften the language explaining RFC9345's recommended validity period.